pkg/validatingadmissionpolicy/builder.go

```diff
@@ -105,12 +105,22 @@ func BuildValidatingAdmissionPolicyBinding(
 
 	// set validation action for vap binding
 	var validationActions []admissionregistrationv1alpha1.ValidationAction
-	action := cpol.GetSpec().GetValidationFailureAction()
-	if action.Enforce() {
-		validationActions = append(validationActions, admissionregistrationv1alpha1.Deny)
-	} else if action.Audit() {
-		validationActions = append(validationActions, admissionregistrationv1alpha1.Audit)
-		validationActions = append(validationActions, admissionregistrationv1alpha1.Warn)
+	validateAction := cpol.GetSpec().Rules[0].Validation.ValidationFailureAction
```

Comment:
Why it checks the first rule only?

Reply:
Because we don't generate ValidatingAdmissionPolicy unless Kyverno has only one `validate` rule.

Reply:
In case of different rule types, ValidatingAdmissionPolicies aren't generated and a message is displayed in the status of Kyverno policy.